modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -254,31 +260,19 @@ private void addOperationObjectResponseInformation(Map<String, Object> operation
     }
 
     private void addExtraReservedWords() {
-        this.reservedWords.add("BASE_PATH");
-        this.reservedWords.add("BaseAPI");
-        this.reservedWords.add("RequiredError");
-        this.reservedWords.add("COLLECTION_FORMATS");
-        this.reservedWords.add("FetchAPI");
-        this.reservedWords.add("ConfigurationParameters");
-        this.reservedWords.add("Configuration");
-        this.reservedWords.add("configuration");
-        this.reservedWords.add("HTTPMethod");
-        this.reservedWords.add("HTTPHeaders");
-        this.reservedWords.add("HTTPQuery");
-        this.reservedWords.add("HTTPBody");
-        this.reservedWords.add("ModelPropertyNaming");
-        this.reservedWords.add("FetchParams");
-        this.reservedWords.add("RequestOpts");
-        this.reservedWords.add("exists");
-        this.reservedWords.add("RequestContext");
-        this.reservedWords.add("ResponseContext");
-        this.reservedWords.add("Middleware");
-        this.reservedWords.add("ApiResponse");
-        this.reservedWords.add("ResponseTransformer");
-        this.reservedWords.add("JSONApiResponse");
-        this.reservedWords.add("VoidApiResponse");
-        this.reservedWords.add("BlobApiResponse");
-        this.reservedWords.add("TextApiResponse");
+        super.registerReservedWordsCaseInsensitive(
```

Comment:
case sensitive

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)